react/game/src/app/components/withLoading/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import styles from './styles.scss';
+
+const withLoading = (Component, external) => props => {
+  const { loading, msgLoading } = props;
+  const endClass = external ? 'External' : 'Internal';
```

Comment:
I would change this name (External, Internal) for something a bit more descriptive. Maybe something that gives you an idea of what this variable will be used for without having to look at the whole code to understand it. 🙂

Reply:
thanks for your comment kike, you are right!